src/presentation/Routes.js

```diff
@@ -1,9 +1,13 @@
-const { Router }  = require("express");
+const { Router } = require("express");
 
 const router = Router()
 
-const UserController = require("./controllers/UserController");
+const { UserController } = require("./controllers/UserController");
 
 router.get("/", UserController.handle);
 
+router.post("/customer", UserController.postUser);
+
+router.get("/customer", UserController.getUser);
+
```

Comment:
Sugestão: Tudo o que for referente a User, seria melhor colocar dentro de uma outra rota. Ex: UserRoute.js